src/sass/components/_btn.scss

```diff
@@ -0,0 +1,57 @@
+.btn{
+@include font-mixin(inherit, 700, 16px, 1.31,unset, unset);
```

Comment:
Dawid jednak napisał by nie używać inherit w scss, więc zamień na DM Sans
domyślny letter-spancing to normal, popraw
ustaw kolor zmienną